waku/node/v2/wakunode2.nim

```diff
@@ -206,9 +207,10 @@ proc mountFilter*(node: WakuNode) =
   node.switch.mount(node.wakuFilter)
   node.subscriptions.subscribe(WakuFilterCodec, node.wakuFilter.subscription())
 
-proc mountStore*(node: WakuNode) =
+proc mountStore*(node: WakuNode, path: string) =
+  var db = db_sqlite.open(path, "", "", "")
```

Comment:
error handling if it can't find db

Reply:
Why would there be error handling on that? On first time it just creates it. Would one need to explicitly create the db beforehand?

Reply:
I don't know the API, but I would be surprised if no errors can occur here for opening a file. Permissions, no file, folder, busy resource, etc.

Worst case it is an error path that doesnt get hit.